sky/client/cli/command.py

```diff
@@ -3762,12 +3762,14 @@ def jobs():
               default=None,
               type=str,
               help='Recovery strategy to use for managed jobs.')
-@click.option('--priority',
-              type=click.IntRange(0, 1000),
-              default=None,
-              show_default=True,
-              help=('Job priority from 0 to 1000. A higher number is higher '
-                    'priority. Default is 500.'))
+@click.option(
+    '--priority',
+    type=click.IntRange(constants.MIN_PRIORITY, constants.MAX_PRIORITY),
+    default=None,
+    show_default=True,
+    help=
+    f'Job priority from ({constants.MIN_PRIORITY} to {constants.MAX_PRIORITY}).'
+    f' Default: {constants.DEFAULT_PRIORITY}.')
```

Comment:
nit - put the space on the previous line rather than the next
```suggestion
    f'Job priority from ({constants.MIN_PRIORITY} to {constants.MAX_PRIORITY}). '
    f'Default: {constants.DEFAULT_PRIORITY}.')
```